lib/tests/form_method-issue-45.rs

```diff
@@ -24,11 +24,24 @@ use rocket::http::ContentType;
 fn method_eval() {
     let rocket = rocket::ignite().mount("/", routes![bug]);
 
-    let mut req = MockRequest::new(Patch, "/")
+    let mut req = MockRequest::new(Post, "/")
         .header(ContentType::Form)
         .body("_method=patch&form_data=Form+data");
 
     let mut response = req.dispatch_with(&rocket);
     let body_str = response.body().and_then(|b| b.into_string());
     assert_eq!(body_str, Some("OK".to_string()));
 }
+
+#[test]
+fn method_eval_with_Get_should_fail() {
+    let rocket = rocket::ignite().mount("/", routes![bug]);
+
+    let mut req = MockRequest::new(Get, "/")
+        .header(ContentType::Form)
+        .body("_method=patch&form_data=Form+data");
+
+    let mut response = req.dispatch_with(&rocket);
+    let body_str = response.body().and_then(|b| b.into_string());
```

Comment:
Don't look at the body. We should get a 404, so test for `response.status(), Status::NotFound`. That suffices.